cmd/config/internal/commands/cmdlistsetters_test.go

```diff
@@ -251,8 +251,116 @@ spec:
 `,
 			expected: `  NAME    VALUE   SET BY    DESCRIPTION    COUNT  
   image   nginx   me2      hello world 2   3      
-  SUBSTITUTION    PATTERN      SETTERS    
+  SUBSTITUTION    PATTERN    REFERENCES   
   image          IMAGE:TAG   [image,tag]  
+`,
+		},
+
+		{
+			name: "list array setter",
+			openapi: `
+apiVersion: v1alpha1
+kind: Example
+openAPI:
+  definitions:
+    io.k8s.cli.setters.list:
+      items:
+        type: string
+      maxItems: 3
+      type: array
+      description: hello world
+      x-k8s-cli:
+        setter:
+          name: list
+          value: List Values
```

Comment:
Why would we have both `value` and `listValues` set?

Reply:
Not needed. Removed.